setup/deb-setup.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+echo 'Setting up your developmental environment. This may take a while.'
+
+echo '[*] Adding keys...'
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
```

Comment:
`curl` may be absent. If so, it should exit at this point with instructions to install curl.

Reply:
Should it exit the script or try to install curl also?

Reply:
Either way would be okay.